src/main/cpp/util/port.h

```diff
@@ -71,6 +71,21 @@
 #endif
 
 
+// CAN_FIND_OWN_EXECUTABLE_PATH
+//
+// Indicates that a running process can find a path to its own executable.
+#if !defined(__OpenBSD__)
+#define CAN_FIND_OWN_EXECUTABLE_PATH
+#endif
+
+// HAVE_EMULTIHOP
+//
+// Indicates that errno.h defines EMULTIHOP.
+#if !defined(__OpenBSD__)
+#define HAVE_EMULTIHOP
```

Comment:
Same here as above.

Defining `HAS_NO_EMULTIPHOP=1` allows achieving the same effect as `HAVE_EMULTIPOP` but without making assumptions of other platforms.

Reply:
I realized that we could eliminate `HAVE_EMULTIHOP` entirely. It no longer exists.